src/modules/assetPack/utils.ts

```diff
@@ -1,3 +1,39 @@
+import { RawAssetPack, FullAssetPack } from './types'
+import { RawAssetContents } from 'modules/asset/types'
+import { getContentsCID } from 'modules/asset/utils'
+
 export const MAX_TITLE_LENGTH = 20
 export const MIN_TITLE_LENGTH = 3
 export const MAX_THUMBNAIL_SIZE = 200000 // bytes
+
+export async function rawAssetPackToFullAssetPack(rawAssetPack: RawAssetPack): Promise<[FullAssetPack, RawAssetContents]> {
+  const fullAssetPack: FullAssetPack = { ...rawAssetPack, assets: [] }
+  const rawContents: RawAssetContents = {}
+
+  for (let asset of rawAssetPack.assets) {
+    const { contents } = asset
+    rawContents[asset.id] = {}
+    const newAsset = {
+      ...asset,
+      contents: await getContentsCID(asset)
+    }
+    // save rawContents as "cid": Blob
```

Comment:
Move this comment to the RawAssetContents type

Reply:
On second thought, this is a bit weird:

RawAsset saves content as `path: Blob`, while Asset saves it as `path: cid`
The combination of `cid: Blob` is not used anywhere